website/content/api-docs/system/pprof.mdx

```diff
@@ -75,6 +119,26 @@ $ curl \
     http://127.0.0.1:8200/v1/sys/pprof/heap
 ```
 
+## Mutex
+
+This endpoint returns a sampling of goroutines holding contended mutexes.
+
+It is included for completeness, but since Vault doesn't normally enable
```

Comment:
Same question here. Do we need to satisfy completeness if this is not going to return anything?

Reply:
Addressed in the previous comment. Marking as resolved.